sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -335,7 +336,8 @@ private synchronized ClaimsBasedSecurityNode getOrCreateCBSNode() {
 
     private synchronized Connection getOrCreateConnection() throws IOException {
         if (connection == null) {
-            logger.info("Creating and starting connection to {}:{}", handler.getHostname(), handler.getProtocolPort());
+            logger.info("connectionId[{}] Creating and starting connection to {}:{}", connectionId,
```

Comment:
To follow along with the other formatting.

```suggestion
            logger.info("connectionId[{}]: Creating and starting connection to {}:{}", connectionId,
```